doc/apiv3.apib

```diff
@@ -157,12 +154,330 @@ he is allowed to see his own time entries in projects where the costs module is
             }
 
+# Group Cost Entries
+
+## Linked Properties:
+|  Link       | Description                                 | Type          | Constraints           | Supported operations |
+|:-----------:|-------------------------------------------- | ------------- | --------------------- | -------------------- |
+| self        | This cost entry                             | CostEntry     | not null              | READ                 |
+| project     | The project in which this entry was logged  | Project       | not null              | READ                 |
+| costType    | The type of this entry                      | CostType      | not null              | READ                 |
+| user        | The user logging this entry                 | User          | not null              | READ                 |
+| workPackage | The work package that got the entry logged  | WorkPackage   | not null              | READ                 |
+
+## Properties
+| Property    | Description                                 | Type        | Constraints | Supported operations | Condition                   |
+| :---------: | ------------------------------------------- | ----------- | ----------- | -------------------- | --------------------------- |
+| id          | cost entry id                               | Integer     | x > 0       | READ                 |                             |
+| spentUnits  | The amount of units logged in this entry    | Float       |             | READ                 |                             |
+| spentOn     | The date when the units were spent          | Date        |             | READ                 |                             |
+| createdAt   | Time of creation                            | DateTime    |             | READ                 |                             |
+| updatedAt   | Time of the most recent change to the entry | DateTime    |             | READ                 |                             |
+
+## Cost Entry [/api/v3/cost_entries/{id}]
+
++ Model
+    + Body
+
+            {
+                "_type": "CostEntry",
+                "_links": {
+                    "self": {
+                        "href": "/api/v3/cost_entries/1"
+                    },
+                    "project": {
+                        "href": "/api/v3/projects/1"
+                    },
+                    "costType": {
+                        "href": "/api/v3/cost_types/1"
+                    },
+                    "user": {
+                        "href": "/api/v3/users/1"
+                    },
+                    "workPackage": {
+                        "href": "/api/v3/work_packages/1"
+                    }
+                },
+                "id": 1,
+                "spentUnits": 3.14,
+                "spentOn": "2015-03-31",
+                "createdAt": "2015-03-31T08:51:20Z",
+                "updatedAt": "2015-03-31T08:51:20Z"
+            }
```

Comment:
I usually don't like to include the embedded sections if they merely contain the resolved links.

That adds just duplication and sources of (documentation) errors. Per spec a response is valid if it does not resolve any of the links it provides. (read: `_embedded` *might* be required to specify, but if not I tend to leave it out)

This apparently has another plus side: You do not get the impression that you should rely on something being embedded, when in fact it might be ommited.